### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by adieg**

### CODE REVIEW

File: participants/adieg.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "Name": "Adi Egozi",
+  "linkedin": "Adi Egozi",
```
Comment (szabgab) [inline participants/adieg.json]: This is not your username on linkedin. this `adi-egozi-802674227` is. (and you could get rid of those numbers as well

Comment (szabgab) [reply participants/adieg.json]: See https://szabgab.com/beautify-your-linkedin-profile-link.html

Comment (adieg) [reply participants/adieg.json]: Thanks! I was trying to find some kind of unique identifier in my linkedin and couldn't find it.

Comment (szabgab) [reply participants/adieg.json]: Oh so it was unclear that the URL is the unique id

**Pull request merged**